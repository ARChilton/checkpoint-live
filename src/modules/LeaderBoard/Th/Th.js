import React from 'react'
import PropTypes from 'prop-types'

const Th = ({ headers }) => (
  headers.map(heading => (
    <th>{heading}</th>
  ))
)

Th.propTypes = {
  headers: PropTypes.arrayOf(PropTypes.string).isRequired,
}

export default Th
