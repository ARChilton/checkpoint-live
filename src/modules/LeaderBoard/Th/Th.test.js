import React from 'react'
import renderer from 'react-test-renderer'
import Th from './Th'


it('renders 5 table heads', () => {
  const headers = ['a', 'b', 'c', 'd', 'e']
  const tree = renderer
    .create(<Th headers={headers} />).toJSON()

  expect(tree).toMatchSnapshot()
})
