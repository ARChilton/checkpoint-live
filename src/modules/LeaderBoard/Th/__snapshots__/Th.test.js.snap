// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 5 table heads 1`] = `
Array [
  <th>
    a
</th>,
  <th>
    b
</th>,
  <th>
    c
</th>,
  <th>
    d
</th>,
  <th>
    e
</th>,
]
`;
